linux_os/guide/system/permissions/files/dir_system_commands_group_root_owned/rule.yml

```
@@ -36,11 +36,13 @@ rationale: |-
 severity: medium
 
 identifiers:
+    cce@sle12: CCE-83244-4
     cce@sle15: CCE-85743-3
 
 references:
     disa: CCI-001499
     nist: CM-5(6),CM-5(6).1
+    stigid@sle12: SLES-12-010883
     stigid@sle15: SLES-15-010362
     srgid: SRG-OS-000259-GPOS-00100 
```

Could you fix this while we are at it?
```suggestion
    srg: SRG-OS-000259-GPOS-00100 
```

---

The PR needed a rebase so this should be fixed in latest version, thanks for the feedback :bow:

---

Unfortunately, it is not fixed.

---

we could merge as it is and open a new PR fixing that since he was not actually touching these line... it was something that was previously merged.

---

ok, why not. Merging then.